src/lib/middleware/session.ts

```diff
@@ -15,7 +16,9 @@ export function sessionMiddleware(): RequestHandler {
     domain: process.env.NODE_ENV === "development" ? "" : COOKIE_DOMAIN,
     name: SESSION_COOKIE_KEY,
     maxAge: SESSION_COOKIE_MAX_AGE,
-    secure: process.env.NODE_ENV === "production" || NODE_ENV === "staging",
+    secure:
+      (process.env.NODE_ENV === "production" || NODE_ENV === "staging") &&
```

Comment:
It's weird that we're using both `process.env.NODE_ENV` and `NODE_ENV` here. Should they both be `NODE_ENV` which supports a default?

Reply:
Yes...I'm not even sure we want the option for "staging" anymore.

Reply:
I recall us getting rid of `staging` related stuff a few years back; is it still in use somewhere?

Reply:
Ah, I remember now -- it was removed, then re-added when we wanted to add a banner indicating that we were _on_ staging, so as to not confuse (weird req from random users). That banner is only on old pages and can be killed (we didn't bother migrating to new). So yeah, AFAIK we can remove this env.